x-pack/platform/packages/shared/onechat/onechat-common/tools/tool_result.ts

```diff
@@ -19,7 +19,8 @@ export enum ToolResultType {
 
 export enum SupportedChartType {
   Metric = 'metric',
-  Map = 'map',
```

Comment:
this leaked in when adding original tool ... there is no map support yet, so removing this.